src/components/MessagesList/MessagesGroup/Message/MessageButtonsBar/MessageButtonsBar.vue

```diff
@@ -702,6 +717,29 @@ export default {
 			this.$emit('update:isReactionsMenuOpen', true)
 		},
 
+		async forwardToNote() {
+			let noteToSelf = this.$store.getters.conversationsList.filter(conversation => conversation.type === CONVERSATION.TYPE.NOTE_TO_SELF)?.[0]
+
+			// If note to self doesn't exist, it will be regenerated
+			if (!noteToSelf) {
```

Comment:
I'd move this check to store action also
For example, if token is not provided, default option would be noteToSelf